src/Symfony/Component/Form/Resources/translations/validators.fr.xlf

```diff
@@ -18,6 +18,122 @@
                 <source>This value is not a valid HTML5 color.</source>
                 <target>Cette valeur n'est pas une couleur HTML5 valide.</target>
             </trans-unit>
+            <trans-unit id="100">
+                <source>Please enter a valid birthdate.</source>
+                <target>Veuillez entrer une date de naissance valide.</target>
+            </trans-unit>
+            <trans-unit id="101">
+                <source>The selected choice is invalid.</source>
+                <target>Le choix sélectionné est invalide.</target>
+            </trans-unit>
+            <trans-unit id="102">
+                <source>The collection is invalid.</source>
+                <target>La collection est invalide.</target>
+            </trans-unit>
+            <trans-unit id="103">
+                <source>Please select a valid color.</source>
+                <target>Veuillez sélectionner une couleur valide.</target>
+            </trans-unit>
+            <trans-unit id="104">
+                <source>Please select a valid country.</source>
+                <target>Veuillez sélectionner un pays valide.</target>
+            </trans-unit>
+            <trans-unit id="105">
+                <source>Please select a valid currency.</source>
+                <target>Veuillez sélectionner une devise valide.</target>
+            </trans-unit>
+            <trans-unit id="106">
+                <source>Please choose a valid date interval.</source>
+                <target>Veuillez choisir un intervalle de dates valide.</target>
+            </trans-unit>
+            <trans-unit id="107">
+                <source>Please enter a valid date and time.</source>
+                <target>Veuillez saisir une date et une heure valides.</target>
+            </trans-unit>
+            <trans-unit id="108">
+                <source>Please enter a valid date.</source>
+                <target>Veuillez entrer une date valide.</target>
+            </trans-unit>
+            <trans-unit id="109">
+                <source>Please select a valid file.</source>
+                <target>Veuillez sélectionner un fichier valide.</target>
+            </trans-unit>
+            <trans-unit id="110">
+                <source>The hidden field is invalid.</source>
+                <target>Le champ masqué n'est pas valide.</target>
+            </trans-unit>
+            <trans-unit id="111">
+                <source>Please enter an integer.</source>
+                <target>Veuillez saisir un entier.</target>
+            </trans-unit>
+            <trans-unit id="112">
+                <source>Please select a valid language.</source>
+                <target>Veuillez sélectionner une langue valide.</target>
+            </trans-unit>
+            <trans-unit id="113">
+                <source>Please select a valid locale.</source>
+                <target>Veuillez sélectionner un paramètre régional valide.</target>
```

Comment:
une langue valide?

Reply:
I hesitated because we already have this 'Please select a valid language.', it will be the same!

Reply:
Yeah, This is a tricky one. 

Im okey with it being the same as "valid language". 

I agree with Fabien.

Reply:
yes, updated :)

Reply:
Thank you